azkaban-sql/src/sql/create.execution_flows_status.sql

```diff
@@ -0,0 +1,25 @@
+-- -----------------------------------------------------------------
+-- Create lookup table for 'status' values in execution_flows table
```

Comment:
Consider adding a comment similar to the pull request description so that readers know what this table is for.